traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.4.1
+version: 8.4.2
```

Comment:
Can you set a proper version? We had some PR's merged before so we should bump that version as well

Reply:
Done